src/js/BookReader.js

```diff
@@ -1789,15 +1794,17 @@ BookReader.prototype.prefetchImg = function(index) {
   }
 
   if (loadImage) {
-    var img = document.createElement("img");
-    $(img).addClass('BRpageimage').addClass('BRnoselect');
+    const div = this.leafTemplate({}, index);
+    $('<img />', {
+      'class': 'BRpage-frame BRpageimage',
```

Comment:
BRpage-frame looks unused
```suggestion
      'class': 'BRpageimage',
```